cleanlab_studio/utils/data_enrichment/enrich.py

```diff
@@ -18,7 +18,7 @@ def enrich_data(
     prompt: str,
     data: pd.DataFrame,
     *,
-    regex: Optional[Union[str, re.Pattern[str], List[re.Pattern[str]]]] = None,
+    regex_replacements: Optional[Union[Replacement, List[Replacement]]] = None,
```

Comment:
I'm not sure how closely we want this to mirror what we currently have in `EnrichmentOptions`, however that's recently been switched to be `replacements`.